serde-proto/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "serde-proto"
+version = "0.1.0"
+edition = "2021"
+
+
+[build-dependencies]
+tonic-build = "0.8.4"
+
+[dependencies]
+tracing = "0.1.37"
```

Comment:
Is tracing an tonic/tonic-build needed here?

Reply:
Tracing is not needed. Removed it now.